README.md

```diff
@@ -12,14 +12,53 @@ Here you'll find starter code and more advanced example use cases.
 </div>
```

Comment:
Maybe this can be uncentered and moved before `# Prefect Recipes`?

Reply:
Interesting! I kind of like it as is but I'm no aesthetics expert. @taylor-curran?

Reply:
I don't have strong opinions on this! I'll just close this :P